boards/arm/nucleo_h745zi_q/nucleo_h745zi_q_m7.dts

```diff
@@ -63,7 +62,7 @@
 	div-m = <1>;
 	mul-n = <120>;
 	div-p = <2>;
-	div-q = <2>;
+	div-q = <8>;
```

Comment:
By default the fdcan clock source is HSE
Setting the pll1Q as fdcan clock source is done with :
```
			clocks = <&rcc STM32_CLOCK_BUS_APB1_2 0x00000100>,
				 <&rcc STM32_SRC_PLL1_Q USB_SEL(1)>;
```

Reply:
Actually, the current FDCAN driver for H7 selects specifically the PLL1Q as clock source.
This driver has not yet been updated to use a source clock defined in the device tree, but that would be the best way forward.
It is already done in the can_stm32_fdcan driver, so it should not be too difficult.

Reply:
> Actually, the current FDCAN driver for H7 selects specifically the PLL1Q as clock source.
> This driver has not yet been updated to use a source clock defined in the device tree, but that would be the best way forward.
> It is already done in the can_stm32_fdcan driver, so it should not be too difficult.

True.

Reply:
> By default the fdcan clock source is HSE. 
> Why changing the pll1q if fdcan is still clocked by HSE 
> Setting the pll1Q as fdcan clock source is done with :
> ```
> 			clocks = <&rcc STM32_CLOCK_BUS_APB1_2 0x00000100>,
> 				 <&rcc STM32_SRC_PLL1_Q FDCAN_SEL(1)>;
> ```

See the answer about stm32h7 fdcan diver. Maybe I need here additional PR. But for the moment - this is a fix for this Board and will stay in case fdcan will use pll